### DISCUSSION TITLE
@developer Make it Ghost Client Please Bruh it no 1 hacked Client But Make it Ghost Client

### DISCUSSION THREAD

**Discussion by algrowww** [SkidderMC/FDPClient#894] [General]
Recently My Smp Memeber ask me to share the schreen bro make it ghost then every single person loves it its better than vape and others its became no 1 hacked client if u make it ghost please deeply think on my words Have a nice Day! :D

**Comment by B4mb0u** [comment]
lmaoooooooooooooooooooo

**Comment by ghost** [comment]
Not going to happen

**Reply by ianisadin** [reply]
💀 
bro is hacking on his own smp
